### PR TITLE
Fix typos in comments and add leading spaces

### PR DESCRIPTION
**Pull request by calvinballing**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
I ran cspell locally in order to clean up various typos throughout the codebase.  This PR is specifically focused on fixing typos found in code comments.

This was broken off of the larger https://github.com/kubernetes/kubernetes/pull/121268 to make reviewing/merging easier.

Let me know if it should still be chunked smaller.

The linter identified that some of the comments were missing leading spaces, so I added an additional commit that also corrects this issue throughout the codebase.

#### Which issue(s) this PR fixes:

Fixes # n/a
I can make an issue for this if needed.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by swatisehgal — COMMENTED** [review]
/triage accepted 
/priority important-longterm

/lgtm

File: staging/src/k8s.io/kubelet/pkg/apis/dra/v1alpha3/api.pb.go [hunk 1]
```diff
@@ -482,7 +482,7 @@ const _ = grpc.SupportPackageIsVersion4
 type NodeClient interface {
 	// NodePrepareResources prepares several ResourceClaims
 	// for use on the node. If an error is returned, the
-	// response is ignored. Failures for individidual claims
+	// response is ignored. Failures for individual claims
 	// can be reported inside NodePrepareResourcesResponse.
 	NodePrepareResources(ctx context.Context, in *NodePrepareResourcesRequest, opts ...grpc.CallOption) (*NodePrepareResourcesResponse, error)
```
Comment (bart0sh) [inline staging/src/k8s.io/kubelet/pkg/apis/dra/v1alpha3/api.pb.go:487]: pb.go files are auto-generated and shouldn't be edited. Please, consider changing .proto files instead.

**Comment by calvinballing** [comment]
I removed the changes to the two api.on.gp files, and instead made the changes in the api.proto files

**Comment by bart0sh** [comment]
@calvinballing you can run it in a container, I believe.

**Comment by swatisehgal** [comment]
> I'm working on macOS 10.13, and I haven't been able to get `hack/update-codegen.sh` to run to update the proto files. I may need someone else to do that for me

Happy to help with this: https://github.com/calvinballing/kubernetes/pull/1.

**Comment by calvinballing** [comment]
I'm not seeing what in my changes would be causing the tests to fail.  Maybe someone with more context could take a look?

**Comment by bart0sh** [comment]
@calvinballing 
> This was broken off of the larger https://github.com/kubernetes/kubernetes/pull/121268 to make reviewing/merging easier.

This PR changes even more files than https://github.com/kubernetes/kubernetes/pull/121268
I'd suggest to break it so that amount of related SIGs are 1-2.

**Pull request closed without merging**